src/sentry/static/sentry/app/views/projectDetail/projectCharts.tsx

```diff
@@ -12,9 +12,11 @@ import {
   SectionValue,
 } from 'app/components/charts/styles';
 import {Panel} from 'app/components/panels';
+import CHART_PALETTE from 'app/constants/chartPalette';
 import {t} from 'app/locale';
 import {Organization, SelectValue} from 'app/types';
 import {decodeScalar} from 'app/utils/queryString';
+import theme from 'app/utils/theme';
```

Comment:
should we wrap this component with "withTheme" and use the theme from the props?

Reply:
I wrapped it but I am not sure if it's needed when only using direct colors like red300 and not aliased ones